src/vmm/src/devices/virtio/net/device.rs

```diff
@@ -885,6 +884,11 @@ impl Net {
         // If the limiter is not blocked, continue transmitting bytes.
         {
             self.process_tx()
+                .inspect_err(|e| {
+                    if let DeviceError::InvalidAvailIdx(err) = e {
+                        panic!("{}", err)
+                    }
```

Comment:
Maybe just use match?
```rust
match self.process_tx() {
   Ok() => {},
   Err(InvalidAvailIdx) => panic!
   Err(e) => report_net_event_fail
}

Reply:
as you pointed out in the other comments, there's already a panic in report_net_event_fail, so this inspect_err is pointless (and now removed)